content/en/administrators_guide/run.md

```diff
@@ -0,0 +1,127 @@
+
+It is important to keep an eye on consumption, and equally so, to adopt the tools provided for this purpose. Datadog provides an [estimated usage metrics][5] dashboard that can be utilized to create the foundation of this capability. There are also out of the box dashboards for visualizing the estimated usage across all of your [logs, metrics, and traces][6]. 
+
+### Deploy Dashboards and monitors
+
+After becoming familiar with Datadog, your users may start asking for refinements and adjustments, particularly of the items they interact with the most; [dashboards][7] and [monitors][8]. The dashboards, monitors, SLOs, and other content objects are designed for iterative development.  At their core, these items are written in JSON. They can be cloned, exported, modified, imported, and stored as flat files. Additionally, there is a [terraform provider][9] available, and a [dashboards API][10] that can interact with and create dashboards.  
+
+When creating dashboards, focus on what content to display rather than on the construction process of the dashboard itself. This creative process is embraced within dashboards creation tools and in the dashboards that come with the product. Each of the dashboards included within the {{< translate key="integration_count" >}} integrations is a value-added template for monitoring for the corresponding technology. Out-of-the-box dashboards share with you Datadog's experience and prescriptive model for observability.  
+  
+**Recommendations:**  
+
+- Determine the purpose of the dashboard you're creating.   
+- Set Monitors for Datadog usage built off the [Estimated Usage metrics][6].  
+- Create [anomaly or change monitors][11] on these same estimated usage metrics to send alerts when your Datadog usage spikes.    
+- [Re-use and clone][12] other dashboards to save time.  
+- Utilize [OOTB dashboards][13] to govern consumption.
+
+A common OOTB Dashboard is the AWS EC2 Overview Dashboard:
+
+{{< img src="/administrators_guide/ec2_overview.png" alt="AWS EC2 Overview Dashboard" style="width:90%;">}}
+
+### API key rotation 
+
+The Datadog platform uses standard Restful API Key Authentication, and recommends that standard [API Key Security][14] and rotation. In addition, it is useful to organize the assignment of these keys to logical working groups to optimize the security profile and rotation operation.
+
+**Recommendations:**   
+
+Incorporate Datadog API and App Keys into your own systems for key management. Organize keys into groups that can be easily maintained. 
+
+### RBAC objects roles, teams, and permission sets
+
+Datadog [RBAC][15] is dependent upon your SAML provider, and the AD/LDAP store upstream of that SAML provider. It can mirror the AD user groups, and assign Datadog-specific permissions in a standard group-mapping fashion. Collaboration between Datadog admins and SAML/AD/LDAP admins will be necessary to exchange the specific group names and attributes that go into the key-value structure.    
```

Comment:
```suggestion
Datadog [RBAC][15] relies on your SAML provider, and the AD/LDAP store upstream of that SAML provider. It can mirror the AD user groups and assign Datadog-specific permissions in a standard group-mapping. Collaboration between Datadog admins and SAML/AD/LDAP admins is necessary to exchange the specific group names and attributes for the key-value structure.    
```